src/rpc/blockchain.cpp

```diff
@@ -1156,7 +1156,7 @@ static RPCHelpMan pruneblockchain()
     while (block->pprev && (block->pprev->nStatus & BLOCK_HAVE_DATA)) {
         block = block->pprev;
     }
-    return uint64_t(block->nHeight);
+    return int64_t(block->nHeight) - 1;
```

Comment:
nit, braced init seems to be the current preferred approach
```suggestion
    return int64_t{block->nHeight} - 1;
```